app/models/concerns/download_path.rb

```diff
@@ -3,8 +3,9 @@ def download_path(dsid='content')
     url_helpers = Rails.application.routes.url_helpers
     url_helpers.url_for(:controller => :downloads, 
                         :action => :show, 
-                        :id => pid, 
-                        :datastream_id => dsid, 
+                        :id => id,
+                        # TODO: determine what should replace datastream_id
+                        # :datastream_id => dsid,
```

Comment:
`datastream_id` was a Samvera Hydra thing. I found [this old code snippet](https://github.com/samvera/hydra-head/blob/v9.5.0/hydra-core/app/controllers/concerns/hydra/controller/download_behavior.rb#L54-L69) which says that `datastream_id` is deprecated and people should use `file` (a filename?) instead.

Reply:
Good context to have. Yes, filename should be used. If I remember correctly, I left this here because I wasn't sure what should replace `datastream_id`. Also, is it possible for a file to only have a datastream_id and not have a filename? Seems unlikely, but it won't hurt to write code to handle for that if it is possible.

Reply:
issue #100